.github/workflows/docker.yaml

```
@@ -0,0 +1,39 @@
+name: Docker Image CI
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+jobs:
+  buildx:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+        id: qemu
+
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      # - name: Login to Docker Registry
+      #   uses: docker/login-action@v1
+      #   with:
+      #     username: ${{ secrets.DOCKERHUB_USERNAME }}
+      #     password: ${{ secrets.DOCKERHUB_TOKEN }}
```

```suggestion
      # - name: Login to Docker Registry
      #   uses: docker/login-action@v1
      #   with:
      #     username: ${{ secrets.DOCKERHUB_USERNAME }}
      #     password: ${{ secrets.DOCKERHUB_TOKEN }}
```

@dhui In order to use build and push Docker images automatically, you will need to change these